ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/committer/CommitContext.scala

```diff
@@ -68,9 +67,9 @@ private[kvutils] case class CommitContext(
   /** Generates a collection from the inputs as determined by a partial function.
     * Records all keys in the input as being accessed.
     */
-  def collectInputs[B, That](
```

Comment:
I couldn’t find a nice way of making this work in the previous generality on both 2.12 and 2.13 and we only use it for `Map[K, V]` so I’ve just restricted the type.